client/cli/src/informant.rs

```diff
@@ -40,6 +41,11 @@ pub fn build(service: &impl AbstractService) -> impl futures::Future<Output = ()
 			} else {
 				trace!(target: "usage", "Usage statistics not displayed as backend does not provide it")
 			}
+			trace!(
+				target: "usage",
+				"Subsystems memory [txpool: {}]",
```

Comment:
What unit does it print the value in? I'd also simplify it to:
`[txpool] X Kbytes` or sth` as the `usage` target will give us this kind of details already.

Reply:
For technical traces invisible to user, I'd always put simple bytes

Reply:
Let's add `bytes` then, although I feel it would be more readable as `KB`